lib/eventboss/runner.rb

```diff
@@ -14,6 +14,8 @@ def launch
 
         self_read = setup_signals([:SIGTERM])
 
+        logger.info('Active Listeners:')
```

Comment:
Wouldn't it be 'Included Listeners:'?

Reply:
Active Listeners sounds good to me. I understand it as a list of listeners that are active (being used). Included is kinda misleading since there is also exclusion in config.